_component_design/help-beacon.md

```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
+- **Do not** use the beacon in leiu of UX best practices with regard to affordance.
+- **Do not** use other forms of notification to deliver the context such as automatic tool tips or directional pop-overs
+- **Do not** obscure the item of text label with the beacon.
+
+{% endcolumn %}
 
-<div class="ws-beacons">
-  <div class="ws-beacon-con">
-    <p>Grey:</p>
-    <div class="ds-beacon-con"><span class="ds-beacon ds-tooltip-source temp-beacon" title="<h3 class='ds-tooltip-h3'>New Feature Title</h3><p class='ds-tooltip-p'>A very short explanation of the new feature and something about what it does.</p><div class='ds-btn-group-tooltip'><button class='ds-btn-sm ds-tooltip-btn' onclick='removeTooltips()' type='button'>Learn More</button><button class='ds-btn-sm-sec ds-tooltip-btn' onclick='removeTooltips()' type='button'>Cancel</button></div>"></span></div>
-  </div>
-  <div class="ws-beacon-con">
-    <p>Theme match:</p>
-    <div class="ds-beacon-con"><span class="ds-beacon-cyan ds-tooltip-source temp-beacon" title="<h3 class='ds-tooltip-h3'>New Feature Title</h3><p class='ds-tooltip-p'>A very short explanation of the new feature and something about what it does.</p><div class='ds-btn-group-tooltip'><button class='ds-btn-sm ds-tooltip-btn' onclick='removeTooltips()' type='button'>Learn More</button><button class='ds-btn-sm-sec ds-tooltip-btn' onclick='removeTooltips()' type='button'>Cancel</button></div>"></div>
-  </div>
-  <div class="ws-beacon-con">
-    <p>Theme complement:</p>
-    <div class="ds-beacon-con"><span class="ds-beacon-orange ds-tooltip-source temp-beacon" title="<h3 class='ds-tooltip-h3'>New Feature Title</h3><p class='ds-tooltip-p'>A very short explanation of the new feature and something about what it does.</p><div class='ds-btn-group-tooltip'><button class='ds-btn-sm ds-tooltip-btn' onclick='removeTooltips()' type='button'>Learn More</button><button class='ds-btn-sm-sec ds-tooltip-btn' onclick='removeTooltips()' type='button'>Cancel</button></div>"></div>
-  </div>
 </div>
+
+### Default layout
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+This example shows the multiple states for the beacon. The beacon will animate on a loop to catch the user’s eye. Once the beacon is clicked, it dissapears. The pop-over could generate another beacon to provide a tour experience for more advanced workflows or features.
+
+You can string beacons together, to create a tour experience. In this instance, only one at a time, and be sure you prioritize most important or impactful feature first. Each pop-over should
+provide an opt out at every step in the tour.
+
+Ideally we should track the clicks and drop offs so we can tweak the tour to be more useful and informative.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"For full CSS specifications, view the developer documentation"] [class:"image bg-light border"] %}
```

Comment:
Add a period at the end of the caption as it's a full sentence. Also, I'm not sure if linking works in captions, but I'd try to generate a link to the developer documentation here (if it's ready).

Reply:
updated.